docs/whats-new.md

```diff
@@ -22,6 +22,9 @@ Release date: TBD, Target late Aug, 2018
 
```

Comment:
We should also add your GIF (downsized) to the deck.gl-data repository and add it as a table row in the HTML above.